geonode/base/models.py

```diff
@@ -1278,10 +1276,12 @@ def instance_is_processed(self):
 
     @property
     def is_copyable(self):
-        from geonode.geoserver.helpers import select_relevant_files
-
         if self.resource_type == "dataset":
-            allowed_file = select_relevant_files(get_allowed_extensions(), self.files)
+            from geonode.assets.utils import get_default_asset
+            from geonode.geoserver.helpers import select_relevant_files
```

Comment:
Can we just move on top this imports? This API si widely called. Having this here means re-import them everytime. They are cached, yes but part of them can be still re-imported:
https://stackoverflow.com/a/46662587/7597536

Reply:
that would create a circular reference error